shared/src/types/common/index.ts

```diff
@@ -1,2 +1,3 @@
 export * as network from "./network";
 export * from "./email";
+export * as ExtensionConstants from "./extension";
```

Comment:
```suggestion
export * as extension from "./extension";
```

For consistency, lets export it as extension only